testmodels/src/main/java/com/amplifyframework/testmodels/customprimarykey/Blog.java

```diff
@@ -0,0 +1,172 @@
+package com.amplifyframework.testmodels.customprimarykey;
+
+import androidx.core.util.ObjectsCompat;
+
+import com.amplifyframework.core.model.Model;
+import com.amplifyframework.core.model.annotations.HasMany;
+import com.amplifyframework.core.model.annotations.ModelConfig;
+import com.amplifyframework.core.model.annotations.ModelField;
+import com.amplifyframework.core.model.query.predicate.QueryField;
+import com.amplifyframework.core.model.temporal.Temporal;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import static com.amplifyframework.core.model.query.predicate.QueryField.field;
+
+/** This is an auto generated class representing the Blog type in your schema. */
+@SuppressWarnings("all")
+@ModelConfig(pluralName = "Blogs", type = Model.Type.USER, version = 1)
+public final class Blog implements Model {
+  public static final QueryField ID = field("Blog", "id");
+  public static final QueryField NAME = field("Blog", "name");
+  private final @ModelField(targetType="ID", isRequired = true) String id;
+  private final @ModelField(targetType="String", isRequired = true) String name;
+  private final @ModelField(targetType="Post") @HasMany(associatedWith = "blog", type = Post.class) List<Post> posts = null;
+  private @ModelField(targetType="AWSDateTime", isReadOnly = true) Temporal.DateTime createdAt;
+  private @ModelField(targetType="AWSDateTime", isReadOnly = true) Temporal.DateTime updatedAt;
```

Comment:
where is. the blogOwner field?   I. see it in the schema

Reply:
Updated the schema and the models.